scripts/dockerhost/cleanup-dockerhosts

```diff
@@ -14,5 +13,4 @@ do
     echo "Cleaning $hostname"
     sudo docker exec $hostname service docker stop
     sudo docker rm -f $hostname
-    sudo ip link delete $i-ext
```

Comment:
I think devices don't get removed automatically on container delete. I think to remove veth pair you need to give primary device name. From `start-dockerhosts` looks like that is `$i-int ` , may be that is what is giving the error.